<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KillCardWindow</class>
 <widget class="QDialog" name="KillCardWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel {
color: white;
}</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>571</width>
     <height>321</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="killCardConfirmPage">
    <widget class="QPushButton" name="confirmYesButton">
     <property name="geometry">
      <rect>
       <x>89</x>
       <y>213</y>
       <width>161</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 20pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>Kyllä</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>85</y>
       <width>571</width>
       <height>231</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/images/default.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>571</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/images/otti2.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="confirmNoButton">
     <property name="geometry">
      <rect>
       <x>309</x>
       <y>213</y>
       <width>161</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 20pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>Ei</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>110</y>
       <width>381</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 20pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>Haluatko varmasti lukita kortin</string>
     </property>
    </widget>
    <zorder>label_1</zorder>
    <zorder>label_2</zorder>
    <zorder>confirmNoButton</zorder>
    <zorder>label_3</zorder>
    <zorder>confirmYesButton</zorder>
   </widget>
   <widget class="QWidget" name="killCardPinPage">
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>571</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/images/otti2.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="confirmKillCard">
     <property name="geometry">
      <rect>
       <x>309</x>
       <y>260</y>
       <width>251</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>Lukitse kortti</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cancelKillCard">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>251</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>En haluakaan lukita korttia</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>85</y>
       <width>571</width>
       <height>241</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/images/default.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_pin">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>110</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 20pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>Syötä Pin</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditPin">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>165</y>
       <width>321</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <zorder>label_4</zorder>
    <zorder>label_5</zorder>
    <zorder>cancelKillCard</zorder>
    <zorder>confirmKillCard</zorder>
    <zorder>label_pin</zorder>
    <zorder>lineEditPin</zorder>
   </widget>
   <widget class="QWidget" name="killCardKilled">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>571</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/images/otti2.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>85</y>
       <width>571</width>
       <height>231</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/images/default.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>361</width>
       <height>121</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 18pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>Kortti on lukittu, jos haluat sen uudelleen käyttöön, ole yhteydessä ylläpitoon</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>210</y>
       <width>331</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 20pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
